cadc-util/src/test/java/org/opencadc/persist/EntityTest.java

```diff
@@ -105,9 +105,28 @@ public void testTemplate() {
     
```

Comment:
Is the above comment a placeholder for something?

Reply:
yeah, just easy to cut&paste for new tests... it's in my junit template in my IDE